credentials.json

```diff
@@ -0,0 +1,8 @@
+{
+    "sensitive_user": "sensitive_user",
+    "save_user": "save_user",
+    "password": "test_pw",
+    "host": "ikgpgis.ethz.ch",
+    "port": 5432,
+    "database": "hai"
+}
```

Comment:
Are you keeping this? If so it would be better to include it in a `credentials.json.dist` file to clearly specify that the data in it is example data and avoid pushing the real credentials to Git. You should also include `credentials.json` in the gitignore.